.jenkins

```diff
@@ -64,6 +64,7 @@ pipeline {
                               export ENV_CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS};-DKokkos_ENABLE_SERIAL=ON" && \
                               ctest -VV \
                                 -D CMAKE_OPTIONS="${ENV_CMAKE_OPTIONS}" \
+                                -D OUTPUT_JUNIT_FILE=result_GCC-8.4.0_OpenMP.xml \
```

Comment:
We should just not forward OUTPUT_JUNIT_FILE if the variable is empty. Our current approach in Jenkins shows a test summary even without using it.

Reply:
That also means that we don't have to enforce a higher CMake version.

Reply:
But I can not see any results of the tests pre reporting the JUNIT file.
Or are you saying we would be fine with only the configure and build stage being reported?